### ISSUE TITLE
How to upload this Bot to Teams?

### ISSUE THREAD

**Issue by Mattw11486** [howdyai/botkit#2212]
The bot works just fine in bot framework
and I can get it to work in developer mode if I upload the bot as an app in azure
but since my organization doesn't allow me admin mode to azure my only option is to submit to my org through teams via the manage apps button.

![image](https://user-images.githubusercontent.com/76888925/165893695-f9933015-1680-4562-86ec-c5aebea43dcd.png)

However, in order to do that it requires a manifest schema which I have created but I am still getting an error when trying to upload it to teams.

![image](https://user-images.githubusercontent.com/76888925/165894068-ef110729-570d-4b39-a248-f2abc7e26222.png)

Not sure what I am missing here is my current Schema, if anyone has any examples that have gotten this botkit to work with this that would be great!

`
{
    "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
    "manifestVersion": "1.12",
    "version": "1.0.0",
    "id": "83ee6670-bf5f-4b2d-8089-320a49728469",
    "packageName": "com.microsoft.teams.sdbot",
    "developer": {
      "name": "Matt Wood",
      "websiteUrl": "",
      "privacyUrl": "https://www.teams.com/privacy",
      "termsOfUseUrl": "https://www.teams.com/termsofuse"
    },
    "icons": {
      "color": "NodriSWA.png",
      "outline": "NodriSWA.png"
    },
    "name": {
      "short": "Nodri",
      "full": "Nodri the Service Desk Bot"
    },
    "description": {
      "short": "Nodri",
      "full": "Nodri is a Service Desk Q&A Bot for the Service Desk Teams channel that can be expanded to add more Q&A, added to additional channels, and much more!."
    },
    "accentColor": "#FFFFFF",
    "bots": [
      {
        "botId": "83ee6670-bf5f-4b2d-8089-320a49728469",
        "scopes": [
          "personal",
          "team",
          "groupchat"
        ],
        "supportsFiles": true,
        "isNotificationOnly": false,
        "supportsCalling": true,
        "supportsVideo": true
      }
    ],
    "permissions": [
      "identity",
      "messageTeamMembers"
    ],
    "validDomains": []
  }
`

**Comment by benbrown** [comment]
Instead of uploading the schema, I would use the "App Studio" application inside Teams to set up your bot.

**Comment by Mattw11486** [comment]
Yes unfortunately my company doesn't provide the access to the App Studio App :(
